config-example.json

```diff
@@ -16,6 +16,7 @@
             "url" : "https://www.github.com/YourOrganization/RepoOne.git",
             "ms-between-poll": 10000,
             "exclude-dot-files": true,
+            "display-name": "YourOrganization/RepoOne [no dotfiles]",
```

Comment:
Just an example, because this view of this repo has exclude-dot-files=true.
But maybe it should just be something more obviously displayish, e.g. "RepoOne (dot files not indexed)"?

Reply:
Ah, that makes sense. I agree with that rephrasing. If you don't mind updating it, I'll merge this PR.

Reply:
Done, thanks!